docs/profiling.md

```diff
@@ -18,7 +18,69 @@ Install specific nightly versions of TensorBoard, TensorBoard profiler, TensorFl
 pip install --upgrade tb-nightly==2.5.0a20201203 tbp-nightly==2.4.0a20201203 tf-nightly==2.5.0.dev20201203 tensorboard-plugin-wit==1.7.0
 ```
 
-### Usage
+### Programmatic capture
+
+You can instrument your code to capture a profiler trace via the
+{func}`jax.profiler.start_trace` and {func}`jax.profiler.stop_trace`
+methods. Call `start_trace` with the directory to write trace files to. This
```

Comment:
If you want, you can make this ``` {func}`~jax.profiler.start_trace` ``` which will render as **`start_trace()`** with a hyperlink to the source (the tilde removes the module path).